doc/introduction/configuration.rst

```diff
@@ -69,18 +71,22 @@ The following is an example for using ``store_account`` with defaults:
     import strawberryfields as sf
     sf.store_account("MyToken")
 
-where ``"MyToken"`` contains the user specific authentication token.
+where ``"MyToken"`` contains the user-specific authentication token.
 
-It is advised to execute this code snippet **only once** per configuration. This should be done in the same directory where the SF project can be found, separately from executing any other Strawberry Fields scripts. Using the default options it will store the account in the *current working directory* by creating a ``config.toml`` file.
+It is advised to execute this code snippet **only once**, separately any other Python scripts.
```

Comment:
Still seems to be a grammatical omission in this sentence. Anyway I still wasn't 100% happy with it, so I gave a go at modifying it
```suggestion
Typically, a user should only ever have to execute this code snippet once, when initially configurating their system to connect to the Xanadu cloud platform.
```

Reply:
Thank you! I've added this part to the warning message